src/probnum/diffeq/odefiltsmooth/ivpfiltsmooth.py

```diff
@@ -34,61 +33,39 @@ def __init__(self, ivp, gaussfilt):
         """
         if not issubclass(type(gaussfilt.dynamicmodel), ODEPrior):
             raise ValueError("Please initialise a Gaussian filter with an ODEPrior")
-        self.ivp = ivp
         self.gfilt = gaussfilt
+        self.ssqest = 0.0
+        self.with_smoothing = with_smoothing
+        super().__init__(ivp)
+
+    def pre_accepted_callback(self, time, current_guess, current_error):
+        """Update the sigma-squared (ssq) estimate."""
+        self.ssqest = self.ssqest + (self.ssq - self.ssqest) / self.num_steps
```

Comment:
> Maybe also initialize self.ssq to zero in __init__

would this make pylint stop complaining in the future? If so, I find it a good idea (I think I find it a good idea anyway)

I wasnt aware about potential pylint errors because I forgot about all the ignores...

Reply:
It might be currently disabled anyways so maybe CI wouldn't complain about it, but it is definitely something I once saw pylint complain about somewhere in our code.